lib/rex/proto/gss/kerberos/message_encryptor.rb

```diff
@@ -70,6 +75,15 @@ def decrypt_and_verify(data)
           #
           attr_accessor :use_acceptor_subkey
 
+          #
+          # [Boolean] Whether this encryptor will be used for DCERPC purposes (since the behaviour is subtly different)
+          # See MS-KILE 3.4.5.4.1 for details about the exception to the rule:
+          # https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-kile/e94b3acd-8415-4d0d-9786-749d0c39d550
+          #
+          # "For [MS-RPCE], the length field in the above pseudo ASN.1 header does not include the length of the concatenated data if [RFC1964] is used."
+          #
+          attr_accessor :is_dcerpc
```

Comment:
It seems like it would be more accurate and inline with both the [MS-KILE](https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-kile/190ab8de-dc42-49cf-bf1b-ea5705b7a087) documentation and the [RFC](https://www.rfc-editor.org/rfc/rfc4757#section-7.1) to refer to this both here and elsewhere as `dce_style` instead of `is_dcerpc`.